iDiff/cmd/iDiff.go

```diff
@@ -34,67 +32,22 @@ func validateArgs(args []string) (bool, error) {
 	if validArgNum, err := checkArgNum(args); !validArgNum {
 		return false, err
 	}
-	if validArgType, err := checkArgType(args); !validArgType {
-		return false, err
-	}
 	return true, nil
 }
 
 func checkArgNum(args []string) (bool, error) {
 	var errMessage string
 	if len(args) < 3 {
-		errMessage = "Too few arguments. Should have three: [IMAGE ID] [IMAGE ID] [DIFFER]."
+		errMessage = "Too few arguments. Should have three: [IMAGE] [IMAGE] [DIFFER]."
 		return false, errors.New(errMessage)
 	} else if len(args) > 3 {
-		errMessage = "Too many arguments. Should have three: [IMAGE ID] [IMAGE ID] [DIFFER]."
+		errMessage = "Too many arguments. Should have three: [IMAGE] [IMAGE] [DIFFER]."
 		return false, errors.New(errMessage)
 	} else {
 		return true, nil
 	}
 }
 
-func checkArgType(args []string) (bool, error) {
```

Comment:
I see that the tar/dir concept has been abstracted from each other to allow for tars to be used as input.  Adding path validation for the tar input might be useful for the checks as well